.github/workflows/test_template.yml

```
@@ -5,7 +5,7 @@ on: push
 env:
   OTP_VERSION: 24.0.4
   ELIXIR_VERSION: 1.12.2
```

since we're upgrading phoenix, should we upgrade Elixir & OTP to the latest version?
```suggestion
  OTP_VERSION: 24.2.1
  ELIXIR_VERSION: 1.13.3
```

---

I think we should do that in a separate PR @longnd as this PR is only supporting Phoenix 1.6.x, and also Phoenix 1.6.x work with Erlang 24.0.4 and Elixir 1.12.2, we no need to introduce that in a same PR.

---

we can create an issue to upgrade Erlang and Elixir version after merging this PR, what do you think @longnd

---

I have created another PR for that https://github.com/nimblehq/elixir-templates/pull/169 on the new Elixir version, they released a new ExUnit coverage algorithm so the test coverage is reduced, we need to spend time to fix that FYI @longnd, but with the Elixir 1.12.2, everything is good to go.